beta-src/src/components/map/WDMapController.tsx

```diff
@@ -162,6 +162,7 @@ const WDMapController: React.FC = function (): React.ReactElement {
       dispatch(
         gameApiSliceActions.updateTerritoriesMeta(getTerritoriesMeta(data)),
       );
+      dispatch(gameApiSliceActions.updateUnitsDisbanding());
```

Comment:
you don't need this as it is already in the `drawOrders` function right? The `drawOrders` function runs every time `updateOrdersMeta` does. The `updateOrdersMeta` function is already being dispatch in the use effect above this one.

Reply:
Not sure why this is resolved.

Reply:
I removed it from mapcontroller so its resolved